From 9682928239fccb4a9a611d24af9fa1a97fe4951b Mon Sep 17 00:00:00 2001
From: Yu Jiahua <yujiahua1@huawei.com>
Date: Thu, 19 Aug 2021 19:45:28 -0800
Subject: [PATCH 3314/4585] [Huawei] sched: Add menuconfig option for
 CONFIG_SCHED_OPTIMIZE_LOAD_TRACKING

Offering: HULK
hulk inclusion
category: bugfix
bugzilla: 175551

--------------------------------

Move CONFIG_SCHED_OPTIMIZE_LOAD_TRACKING into menu "Scheduler features"

Signed-off-by: Yu Jiahua <yujiahua1@huawei.com>
Reviewed-by: Chen Hui <judy.chenhui@huawei.com>
Signed-off-by: Chen Jun <chenjun102@huawei.com>
Patchwork Links: http://patchwork.huawei.com/project/hulk5.10/list/?series=18137

Conflicts:
	init/Kconfig
Signed-off-by: Zheng Zucheng <zhengzucheng@huawei.com>
---
 init/Kconfig | 15 ++++++++-------
 1 file changed, 8 insertions(+), 7 deletions(-)

diff --git a/init/Kconfig b/init/Kconfig
index 8112931ae91b..9d7056dfa04a 100644
--- a/init/Kconfig
+++ b/init/Kconfig
@@ -774,6 +774,14 @@ config GENERIC_SCHED_CLOCK
 
 menu "Scheduler features"
 
+config SCHED_OPTIMIZE_LOAD_TRACKING
+	bool "Optimize scheduler load tracking"
+	default n
+	help
+	  Optimize scheduler load tracking, when load balance is not important
+	  in system, we close some load tracking in tick and enqueue or dequeue
+	  task, in this way, we can save some unnecessary cpu overhead.
+
 config UCLAMP_TASK
 	bool "Enable utilization clamping for RT/FAIR tasks"
 	depends on CPU_FREQ_GOV_SCHEDUTIL
@@ -2134,13 +2142,6 @@ endmenu		# General setup
 
 source "arch/Kconfig"
 
-config SCHED_OPTIMIZE_LOAD_TRACKING
-	bool "Optimize scheduler load tracking"
-	default n
-	help
-	  Optimize scheduler load tracking, when load balance is not important
-	  in system, we close some load tracking in tick and enqueue or dequeue
-	  task, in this way, we can save some unnecessary cpu overhead.
 
 config RT_MUTEXES
 	bool
-- 
2.17.1

